feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cx-sticky-menu">
+  <div class="cx-menu-bar">
+    <button role="tab" [class.active]="!showFilter" (click)="onMenu()">
```

Comment:
The active tab has to be marked for screen-readers with aria-selected.
```suggestion
    <button role="tab" [attr.aria-selected]="!showFilter" [class.active]="!showFilter" (click)="onMenu()">
```